### ISSUE TITLE
Ho to get back to full stock

### ISSUE THREAD

**Issue by tkso1997** [geekman/xqrepack#8]
Thanks a lot for this wonderful script!

What do I have to do to transform the router back to full stock?
Is a manual upgrade with the stock firmware image sufficient or do I have to take care of something?

**Comment by geekman** [comment]
Yes. Use the web UI and "upgrade" to a stock firmware image. After a reboot, it should then be running the stock firmware without any modifications or SSH access.

**Comment by tkso1997** [comment]
Fantastic! Thanks a lot!
